lib/src/orchestrator/analysis/analysis_controller.dart

```diff
@@ -71,8 +75,25 @@ class AnalysisController {
     return true;
   }
 
+  Future<void> pause() async {
```

Comment:
I think there might be some confusions between pause()/stop() and resume()/start().

From my understanding: start() and stop() are meant to be used by the user while pause() and resume() are linked to the lifecycle of the app?

Reply:
I don't really why user could not just call pause actually.

Reply:
It only cancels the streamSubscription, not the stream itself since it's from the native side.
Even if one calls stop(), they can restart easily by just calling start() since it just reopen a subscription to the native stream.
That may be an error?
Should we stop the native stream as well?

Reply:
effectively we should also stop on native side when app goes background. 
This should improve batterie consumption.

Reply:
I double checked how it's done on the native side, and I think it's done correctly in fact.
The EventChannel destroys the stream in onCancel if needed and that's all that should be done I believe.

Since there is no need to change stop() or start(), I still think that start() and resume() are confusing, same for stop() and pause().

I understand the added value of the code from pause() and resume() (if there is a pending analysis, it is paused and resumed automatically when the app goes in background), but from a user point of view I think that they are confusing.

Maybe a `onAppLifecycleUpdate(appLifeccyel)` could replace resume() and pause() ?

If an user wants to "pause/stop" the image analysis with `pause()`, then the app goes in background and comes back to foreground, the analysis stream will be resumed although it was explicitly paused by the user.